fpdcc/ccfp-asset-dashboard
# Add notes field to phase, small styling change

## Overview

This adds a non-required notes field to phases, and changes the styling of the inputs above it to display better on small screen sizes.

Closes #256

### Demo

![image](https://github.com/fpdcc/ccfp-asset-dashboard/assets/114717958/14084172-e448-455d-b8f8-c6f3770e65b8)

## Testing Instructions
* Create or navigate to a phase
* Confirm that a new notes field appears under the existing inputs
* Enter some text and save
* Confirm that the text persists
